deploy/host-operator/dev/toolchainconfig.yaml

```diff
@@ -6,6 +6,10 @@ spec:
   host:
     automaticApproval:
       enabled: true
+    deactivation:
+      deactivationDomainsExcluded: '@redhat.com'
+    registrationService:
+      environment: 'e2e-tests'
```

Comment:
Should be "dev"

Reply:
Thanks for catching that. Fixed: e77ad7b91dee73517ee703a8773f849d51aee1f4